src/Cartography/Cartography.DynamicMap/Cartography.DynamicMap.csproj

```diff
@@ -38,9 +38,7 @@
 
 	<ItemGroup Condition="'$(TargetFramework)' == 'xamarinios10'">
 		<PackageReference Include="Xamarin.Google.iOS.Maps" Version="3.9.0" />
-		<PackageReference Include="Xamarin.Google.Maps.Utils">
-		  <Version>1.3.4</Version>
-		</PackageReference>
+		<PackageReference Include="Xamarin.Google.Maps.Utils"><Version>1.3.4</Version></PackageReference>
```

Comment:
Should be <PackageReference Include="Xamarin.Google.Maps.Utils" Version="1.3.4" />